src/Resource/CloudAccount/Endpoint.php

```diff
@@ -100,4 +100,22 @@ protected function _waitForPhpVersion(
       return $entity->get('php_version') === $version;
     };
   }
+
+  /**
+   * Clear Nginx Cache
+   *
+   * @return Entity $this
+   * @throws ResourceException If endpoint not available
+   * @throws ApiException If request fails
+   */
+  public function clearNginxCache() : Entity {
```

Comment:
needs to take the Entity as an arg.
```php
public function clearNginxCache(Entity $entity) : Endpoint {
  ...
```

Also, returns `Endpoint` as noted above